File: java-api-core/test/com/kinvey/java/model/BaseUserLookupTest.java
package com.kinvey.java.model;

import junit.framework.TestCase;

/**
 * Created by edward on 7/31/15.
 */
public class BaseUserLookupTest extends TestCase {

    public UserLookup ul;

    public void testConstructor() {
        ul = new UserLookup();
        assertNull(ul.getEmail());
    }

    public void testProperties(){
        ul = new UserLookup();
        ul.setId("id");
        assertEquals("id", ul.getId());
        ul.setEmail("email");
        assertEquals("email", ul.getEmail());
        ul.setFirstName("first");
        assertEquals("first", ul.getFirstName());
        ul.setLastName("last");
        assertEquals("last", ul.getLastName());
        ul.setFacebookID("facebook");
        assertEquals("facebook", ul.getFacebookID());
        ul.setUsername("username");
        assertEquals("username", ul.getUsername());

    }



}

File: java-api-core/test/com/kinvey/java/model/BaseUserLookupTest.kt
package com.kinvey.java.model

import junit.framework.TestCase

/**
 * Created by edward on 7/31/15.
 */
class BaseUserLookupTest : TestCase() {

    var ul: UserLookup? = null

    fun testConstructor() {
        ul = UserLookup()
        assertNull(ul?.email)
    }

    fun testProperties() {
        ul = UserLookup()
        ul?.id = "id"
        assertEquals("id", ul?.id)
        ul?.email = "email"
        assertEquals("email", ul?.email)
        ul?.firstName = "first"
        assertEquals("first", ul?.firstName)
        ul?.lastName = "last"
        assertEquals("last", ul?.lastName)
        ul?.facebookID = "facebook"
        assertEquals("facebook", ul?.facebookID)
        ul?.username = "username"
        assertEquals("username", ul?.username)
    }
}